yt_dlp/YoutubeDL.py

```diff
@@ -2338,7 +2338,7 @@ def _check_formats(formats):
                 return
 
             for f in formats:
-                if f.get('has_drm'):
+                if f.get('has_drm') or f.get('__needs_testing'):
```

Comment:
Should this be a documented field for other extractors to use?

In the case of youtube, IMO it works better as a private field that is sanitized from the info json, since the "needs testing" window is <24 hours

Reply:
its better off as a private field

Reply:
if its private/internal only then it can be removed at any time without compat, so don't document it imo.

When we want to make it official we can look into changing the field name.